environment.mac.yml

```diff
@@ -180,7 +182,11 @@ dependencies:
   - wheel=0.33.1
   - wrapt=1.11.1
   - xarray=0.12.1
+<<<<<<< HEAD
```

Comment:
This looks leftover from a merge conflict

Reply:
We also need to check if this works with our current environment file, which has changed a lot.

~~It might make sense to redo this with whats currently in master~~ It looks like you have already merged master into this; if it works thats good for me